app/controllers/sales_logs_controller.rb

```diff
@@ -40,6 +40,28 @@ def edit
     end
   end
 
+  def destroy
+    @log = SalesLog.visible.find_by(id: params[:id])
+
+    render_not_found and return unless @log
+
+    authorize @log, policy_class: LogPolicy
```

Comment:
if you create the empty classes:
-  `LettingsLogPolicy < LogPolicy`
-  `SalesLogPolicy < LogPolicy`
you won't have to explicitly set the `policy_class` every time